internal/pkg/cli/flag.go

```diff
@@ -181,6 +182,8 @@ Cannot be specified with --%s or --%s.`, dockerFileFlag, dockerFileContextFlag)
 Cannot be specified with --%s.`, imageFlag)
 	dockerFileContextFlagDescription = fmt.Sprintf(`Path to the Docker build context.
 Cannot be specified with --%s.`, imageFlag)
+	sourcesFlagDescription = fmt.Sprintf(`List of paths to source directories or files for a Static Site.
```

Comment:
nit: can we say 
```suggestion
	sourcesFlagDescription = fmt.Sprintf(`List of relative paths to source directories or files for a Static Site.
```
Perhaps we can even do:
```suggestion
	sourcesFlagDescription = fmt.Sprintf(`List of relative paths to source directories or files.
```
Since the "must be specified" already mentions Static Site